sdk-base/src/main/java/com/mapbox/common/MapboxMapsAndroidLogger.kt

```diff
@@ -1,6 +1,6 @@
 package com.mapbox.common
 
-internal object MapboxCommonLogger {
+internal object MapboxMapsAndroidLogger {
```

Comment:
For more context, renaming this internal class because common SDK introduced the same class in the same package, renaming wouldn't be an issue since this is an internal change.

Reply:
And are we going to have this change in maps-android separately from this PR? Cause we released CommonSDK 23.1.0-beta.1 and 23.1.0-beta.2 coming soon... There may be issues with adoption

Reply:
we will rename this class when we do gl-native/common dependency bump, so shouldn't be an issue.